README.md

```diff
@@ -1,73 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Anime-World
 
-> Description the project.
+> "Anime-World" is a website show list of animes and show different facts about them.
 
+[Live Demo](scintillating-khapse-6ebe08.netlify.app)
 
-## Built With
+[Video Demo](https://www.loom.com/share/8302a8a49d7a493388745ddbfdaf65bf)
```

Comment:
- [ ] Kindly make sure your presentation video contains a highlight about a piece of the code the student finds interesting or is proud of making not only the page itself.